packages/babel-plugin-named-asset-import/index.spec.js

```diff
@@ -0,0 +1,38 @@
+import pluginTester from 'babel-plugin-tester';
+import namedAssetImport from './index';
+
+pluginTester({
+  plugin: namedAssetImport,
+  pluginOptions: {
+    loaderMap: {
+      svg: {
+        ReactComponent: '@svgr/webpack?-prettier,-svgo![path]',
+      },
+    },
+  },
+  pluginName: 'named-asset-import',
+  snapshot: false,
+  tests: {
+    defaultImport: {
+      code: 'import foo from "foo";',
```

Comment:
Let's use `logo` instead of `foo` for the tests for consistency. Also I think this file should be named `index.test.js`.

Reply:
tests have been update, file renamed